quacc/calculators/qchem.py

```diff
@@ -0,0 +1,132 @@
+"""
+A wrapper around ASE's Q-Chem calculator that makes it better suited for high-throughput DFT.
+"""
+from __future__ import annotations
+
+import inspect
+import os
+import struct
+import warnings
+
+import ase.units
+import numpy as np
+from ase import Atoms
+from ase.calculators.calculator import FileIOCalculator
+from monty.io import zopen
+from pymatgen.core import Molecule
+from pymatgen.io.ase import AseAtomsAdaptor
+from pymatgen.io.qchem.inputs import QCInput
+from pymatgen.io.qchem.outputs import QCOutput
+from pymatgen.io.qchem.sets import ForceSet
+
+from quacc import SETTINGS
+from quacc.custodian import qchem as custodian_qchem
+
+
+class QChem(FileIOCalculator):
+    """
+
+    Parameters
+    ----------
+    input_atoms
+        The Atoms object to be used for the calculation.
+    use_custodian
+        Whether to use Custodian to run Q-Chem.
+        Default is True in settings.
+    **kwargs
+        Additional arguments to be passed to the Q-Chem calculator. Takes all valid
+        ASE calculator arguments, in addition to those custom to Quacc.
+
+    Returns
+    -------
+    Atoms
+        The ASE Atoms object with attached Q-Chem calculator.
+    """
+
+    def __init__(
+        self,
+        input_atoms: Atoms,
+        charge: None | int = None,
+        spin_multiplicity: None | int = None,
+        qchem_input_params: dict = {},
```

Comment:
Initializing a kwarg with an empty dict isn't recommended. You should set `dict = None` and then in the code within the `__init__` do `self.qchem_input_params = qchem_input_params or {}`.

Reply:
Thanks for taking care of this!